src/metadata_export.py

```diff
@@ -382,7 +382,7 @@ def save_release(self, data_record, node_type, node_id, crdc_id):
                              }], 
                 STUDY_ID: data_record.get(STUDY_ID) or self.submission.get(STUDY_ID)
             }
-            if data_record.get(S3_FILE_INFO) and data_record.get(S3_FILE_INFO).get(FILE_NAME):
+            if self.submission_type != "Metadata Only" and data_record.get(S3_FILE_INFO) and data_record.get(S3_FILE_INFO).get(FILE_NAME):
```

Comment:
Please use constant SUBMISSION_DATA_TYPE_METADATA_ONLY instead of string literal.